### PR TITLE
Fix "Location Tracking Notification" on android

### PR DESCRIPTION
**Pull request by darkerthanblackoff**
## Description

Fixes issue #50 where on Android missed notification that location tracking was disabled.

## How to test

Toggle on/off Location Service in settings.

### CODE REVIEW

File: app/services/LocationService.js [hunk 1]
```diff
@@ -246,8 +246,17 @@ export default class LocationServices {
       } else {
         BackgroundGeolocation.start(); //triggers start on start event
 
-        // TODO: We reach this point on Android when location services are toggled off/on.
-        //       When this fires, check if they are off and show a Notification in the tray
+        BackgroundGeolocation.checkStatus(({ locationServicesEnabled }) => {
+          if (!locationServicesEnabled) {
+            PushNotification.localNotification({
+              id: '55',
```
Comment (tstirrat) [inline app/services/LocationService.js:252]: What is this number?

Comment (darkerthanblackoff) [reply app/services/LocationService.js:252]: This is random number to identify notification for ability cancel it from code.

Comment (tstirrat) [reply app/services/LocationService.js:252]: OK, please make a meaningful `const NOTIFICATION_X = 'blah'` for it

**Comment by tstirrat** [comment]
your history got broken, `git merge develop` does not fix it.. try recreating it with `git rebase -i develop`

**Pull request merged**